### PR TITLE
Fuatue/user-status:  account status capabilities 

### PR DESCRIPTION
**Pull request by KianMiridoozini**
Account status has now a meaningful impact on a users capabilities.

Suspended accounts can not add new cocktails.
Suspended accounts can not add comments or reply to other comments

Banned accounts can not log in

### CODE REVIEW

File: app/views/cocktails/view.php [hunk 1]
```diff
@@ -1,6 +1,8 @@
 <?php
 include __DIR__ . '/../layout/header.php';
 include_once __DIR__ . '/../../helpers/helpers.php';
+$authController = new AuthController();
+$currentUser = $authController->getCurrentUser();
```
Comment (jayb2302) [inline app/views/cocktails/view.php]: I think that this should be in index.php or header.php

**Review by jayb2302 — CHANGES_REQUESTED** [review]
#### Move 
```php
$authController = new AuthController();
$currentUser = $authController->getCurrentUser();
```
from cocktails/view.php to layout/header.php
#### This change will:
- Make $currentUser accessible across views
- Reduce code duplication
- Keep cocktails/view.php focused on display logic.

File: app/views/cocktails/view.php (hunk 1 above)
Comment (KianMiridoozini) [reply app/views/cocktails/view.php]: the issue has been addressed in the latest commit:

**Review by jayb2302 — APPROVED** [review]
Approved ✅

**Pull request merged**